rest/models/quotes.go

```diff
@@ -42,100 +38,25 @@ type QuotesResponse struct {
 
 // ListQuotesParams is the set of path and query parameters that are used when requesting quotes via the ListQuotes method.
 type ListQuotesParams struct {
-	Ticker      string
-	QueryParams ListQuotesQueryParams
-}
-
-// ListQuotesQueryParams is the set of query parameters that can be used when requesting quotes via the ListQuotes method.
-type ListQuotesQueryParams struct {
-	TimestampEQ  *time.Time
-	TimestampLT  *time.Time
-	TimestampLTE *time.Time
-	TimestampGT  *time.Time
-	TimestampGTE *time.Time
-
-	Order *Order
-
-	Limit *int
-
-	Sort *Sort
+	Ticker string `validate:"required"`
 
-	AfterPrimary   *string
```

Comment:
I'd like to know if there's any reason why we should keep `AfterPrimary` `AfterSecondary` and `Cursor`

Reply:
yeah, i'm not sure but it's something we can discuss this afternoon